Unix/cloc

```diff
@@ -8829,6 +8829,10 @@ sub normalize_file_names {                   # {{{1
                 $F_norm = lc "$cwd/$F_norm";
             }
         }
+        # Remove trailing / so it does not interfere with further regex code 
+        # that does not expect it
+        $F_norm =~ s{/$}{}g;
```

Comment:
1. Don't need 'g' modifier here since there can be only 1 trailing slash (/$).
2. The regex will still leave all slashes but the final one in place (e.g. 'abc//' will become 'abc/').